pkg/integrations/v2/app_o11y_receiver/exporters/exporter.go

```diff
@@ -1,11 +1,13 @@
 package exporters
 
 import (
+	"context"
+
 	"github.com/grafana/agent/pkg/integrations/v2/app_o11y_receiver/models"
 )
 
 // AppO11yReceiverExporter is the base interface
 type AppO11yReceiverExporter interface {
 	Name() string
-	Export(payload models.Payload) error
+	Export(ctx context.Context, payload models.Payload) error
```

Comment:
I am not a huge fan of this change as it breaks the current API but to be honest, context is a runtime argument and it can be potentially useful to other consumers so 👍